exercises/wordy/wordy_spec.rb

```diff
@@ -0,0 +1,136 @@
+require_relative "./wordy"
+
+RSpec.describe WordProblem do
+  it "can add one to one" do
+    problem = WordProblem.new("What is 1 plus 1?")
+
+    expect(problem.answer).to eq(2)
+  end
+
+  it "can add two to fifty three" do
+    skip "fix this test second - delete the `skip` lines to enable individual tests"
```

Comment:
`skip` is preferable here. See our (RSpec's) own documentation:

https://relishapp.com/rspec/rspec-core/v/3-4/docs/pending-and-skipped-examples/skip-examples

https://relishapp.com/rspec/rspec-core/v/3-0/docs/pending-and-skipped-examples/pending-examples